rate: 20
age_max: 0.5
start_delay: 4.0